packages/ses/README.md

```diff
@@ -726,11 +726,39 @@ instructions in [SECURITY.md][] to report security-sensitive bugs privately.
 
 For non-security bugs, please use the [regular Issues page][SES Issues].
 
+## Ecosystem Compatibility
+
+Most ordinary JavaScript can run without issues in a realm locked down by SES.
+Exceptions are tracked at [issue #576][incompatibility tracking], and almost
+always take the form of assignments that fail because the
+"[override mistake][override mistake]" prevents overwriting properties inherited
```

Comment:
```suggestion
"[override mistake][override mistake]" prevents overriding properties inherited
```
"overwriting" encourages a common misunderstanding that the issue is assignment to the property itself.